### DISCUSSION TITLE
Will there be a version of the plugin for KDE Plasma 6?

### DISCUSSION THREAD

**Discussion by worthymelight** [catsout/wallpaper-engine-kde-plugin#369] [Q&A]
The development of KDE Plasma 5 has been stopped, today (03/06/2024) the latest version of KDE Plasma 5.27.11 was released, now KDE is developing Plasma 6. I think it would be very good if support for KDE Plasma 6 appeared

**Comment by fxzjshm** [comment]
Check out the qt6 branch: https://github.com/catsout/wallpaper-engine-kde-plugin/tree/qt6
